changedetectionio/html_tools.py

```diff
@@ -53,7 +53,11 @@ def element_removal(selectors: List[str], html_content):
 def xpath_filter(xpath_filter, html_content, append_pretty_line_formatting=False):
     from lxml import etree, html
 
-    tree = html.fromstring(bytes(html_content, encoding='utf-8'))
+    try:
+        tree = etree.HTML(html_content)
+    except:
```

Comment:
So the fix here is to use .HTML parser, and if that fails, fall through to .fromstrong right? is that whats going on?

Reply:
Also, do not change the HTML file added for the test purpose. Because its unique Unicode encoding can be contaminated by your system encoding or IDE encoding.

Reply:
The test file is from https://github.com/dgtlmoon/changedetection.io/issues/1044#issuecomment-1301342481

Reply:
See also [Parsing HTML](https://lxml.de/parsing.html) of lxml.
> The support for parsing broken HTML depends entirely on libxml2's recovery algorithm. It is not the fault of lxml if you find documents that are so heavily broken that the parser cannot handle them. There is also no guarantee that the resulting tree will contain all data from the original document. The parser may have to drop seriously broken parts when struggling to keep parsing. Especially misplaced meta tags can suffer from this, which may lead to encoding problems.